### ISSUE TITLE
Upgrade to Laravel 8 not possible due to "old" guzzlehttp/guzzle

### ISSUE THREAD

**Issue by Tklaversma** [bunq/sdk_php#204]
## Steps to reproduce:
1. Upgrade to Laravel 8

## What should happen:
1. Upgrade should work when bunq updates to ```"guzzlehttp/guzzle" : ^7.0```

## What happens:
1. Upgrade not possible.

## SDK version and environment
- Tested on [1.14.1](https://github.com/bunq/sdk_php/releases/tag/1.14.1)
- [ ] Sandbox
- [X] Production

## Extra info:
If bunq updates their project to use ```"guzzlehttp/guzzle" : ^7.0```, we can upgrade to Laravel 8

**Comment by petermeester** [comment]
I've the same problem. Please fix this issue for Laravel users

**Comment by msubotovic** [comment]
We also face the same problem, we can't use bunq sdk in our project.

**Comment by WouterFlorijn** [comment]
Yes, this is very much blocking for us and should be very simple to fix.

**Comment by RoodFruit** [comment]
Quick **dirty** workaround `composer update --ignore-platform-reqs`

**Comment by RoodFruit** [comment]
Bunq you can close this.

**Comment by two-trick-pony-NL** [comment]
👋 Hey @Tklaversma 

I work at bunq and am currently reviewing the bunq API docs, open issues and SDK’s and came across your issue. I know it’s been open for a while, and I’d love to understand how it’s impacting you. Beyond the obvious -fixing this issue-, what else could we do to improve your experience with our API?

If you want to share your thoughts on how we can improve on our public API:

📅 I'd love to schedule a 15 minute interview to understand your needs through: [Calendly](https://calendly.com/pvandoorn-bunq/30min)
💬 Or if you don't have time for a call, you can give us some general feedback through this [form](https://bunqsurvey.typeform.com/to/xMu9sEfw)

Your feedback will go straight to our product team! 🚀

**Comment by two-trick-pony-NL** [comment]
Hi! We're closing this issue as it's tied to an older version of our SDKs, and we've since completely revamped them as part of bunq Update 28.

This is a fresh start—we've tested the new SDKs extensively, and we’ll be putting them to the test again during our upcoming hackathon this Friday, where we’ll fix idiosyncrasies on the spot. (You’re welcome to [join us](https://www.bunq.com/en-nl/hackathon)!)

If this issue (or a new one) shows up with the updated SDKs, feel free to reopen or file a new report—we're happy to jump on it.
